va/va.h

```diff
@@ -864,14 +881,14 @@ VAStatus vaQueryConfigAttributes (
 /**
  * Contexts and Surfaces
  *
- * Context represents a "virtual" video decode pipeline. Surfaces are render 
- * targets for a given context. The data in the surfaces are not accessible  
- * to the client and the internal data format of the surface is implementatin 
- * specific. 
+ * Context represents a "virtual" video decode, encode or video processing
+ * pipeline. Surfaces are render targets for a given context. The data in the
+ * surfaces are not accessible to the client except derived image is supported
+ * and the internal data format of the surface is implementatin specific.
  *
- * Surfaces will be bound to a context when the context is created. Once
- * a surface is bound to a given context, it can not be used to create  
- * another context. The association is removed when the context is destroyed
+ * Surfaces are provided as a hint of what surfaces will be used when the context
+ * is created. A surface may be shared between different contexts, e.g. a surface
+ * is used as the output of a decode context and the input of a video process context.
```

Comment:
it should describe who and how the surface ownership is maintained, i.e. what happens when a different context uses and destroys them